tests/foreman/cli/test_reporttemplates.py

```diff
@@ -857,3 +857,37 @@ def test_positive_generate_entitlements_report_multiple_formats(self):
             })
             assert vm.hostname in result_csv[1]
             assert self.setup_subs_id[0]['name'] in result_csv[1]
+
+    @tier3
+    def test_positive_schedule_Entitlements_report(self):
+        """Schedule an report using the Entitlements template in csv format.
+
+        :id: 572fb387-86e0-40e2-b2df-e8ec26433610
+
+
+        :setup: Installed Satellite with Organization, Activation key,
+                Content View, Content Host, and Subscriptions.
+
+        :steps:
+            1. hammer report-template schedule --organization '' --id '' --report-format ''
+
+        :expectedresults: report is scheduled and generated containing all the expected information
+                          regarding Entitlements.
+
+        :CaseImportance: High
+        """
+        with VirtualMachine(distro=DISTRO_RHEL7) as vm:
+            vm.install_katello_ca()
+            vm.register_contenthost(self.setup_org['label'], self.setup_new_ak['name'])
+            assert vm.subscribed
+            scheduled_csv = ReportTemplate.schedule({
+                'id': 115,
```

Comment:
What is '115'? Will it always 115? Could we obtain that number dynamically? Would it make sense to add it to constants?

Reply:
115 is the entitlements ID. each report template has a preset ID assigned to it. This should not change.

Reply:
Can we add that into constants? Variable name help to understand.

Reply:
Since each ReportTemplate has its own unique ID, we will still have to specify it anytime we call schedule or report_data. I can look into making it a constant but I don't think it is necessary for these tests. What do you think?